.circleci/config.yml

```diff
@@ -1,114 +1,103 @@
-orbs:
-  architect: giantswarm/architect@0.4.5
-
 version: 2.1
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
-
-    - run: |
-        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/tags/v1.0.0 | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-
-    - run: |
-        date +"%Y" > /tmp/year
-        date +"%m" > /tmp/month
-        date +"%d" > /tmp/day
-    - restore_cache:
-        keys:
-        - go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}-{{ checksum "/tmp/day" }}
-        - go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}
-        - go-cache-v1-{{ checksum "/tmp/year" }}
 
-    - run: ./architect build
-
-    - store_test_results:
-        path: /tmp/results
-
-    - save_cache:
-        key: go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}-{{ checksum "/tmp/day" }}
-        paths:
-        - /tmp/go/cache
-
-    - persist_to_workspace:
-        root: .
-        paths:
-        - ./vault-exporter
-        - ./architect
-
-  deploy:
-    machine: true
-    steps:
-    - checkout
-    - attach_workspace:
-        at: .
-    - deploy:
-        command: |
-            ./architect deploy
+orbs:
+  architect: giantswarm/architect@0.8.15
 
 workflows:
   build:
     jobs:
-      - build
-      - architect/push-to-app-catalog:
-          name: "push-vault-exporter-to-app-catalog"
-          app_catalog: "giantswarm-operations-platform-catalog"
```

Comment:
@corest does vault-exporter still need to be published to giantswarm-operations-platform-catalog? It seems to be still used on gorilla/oby63

See related thread in #team-biscuit Slack channel https://gigantic.slack.com/archives/C010L893ZFW/p1596268659020100

Reply:
Thanks Stevo for rising it. We will be installing vault-exporter from control-plane-catalog there.